### PR TITLE
Ongoing fixes to chart definition and docs

### PR DESCRIPTION
**Pull request by denismakogon**
Adding RAM/CPU resource request/limits to the default chart values in order to make it explicit to users.

### CODE REVIEW

File: fn/values.yaml [hunk 1]
```diff
@@ -58,7 +70,13 @@ fn_lb_runner:
 fn_api:
   image: fnproject/fnserver
   tag: latest
-  resources: {}
+  resources:
```
Comment (devcamcar) [inline fn/values.yaml]: I confirmed that these work, though you may want to add resource requests/limits for ui for the sake of completeness.

Comment (denismakogon) [reply fn/values.yaml]: thanks, will add.

Comment (denismakogon) [reply fn/values.yaml]: done

**Comment by carimura** [comment]
boom

**Pull request merged**